inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
-the mouse button, the annotation is immediately created and you can edit its details in the right
-sidebar.
+* Let‘s examine the right panel:
 
-Congratulations! You have created your first annotation project.
+* {blank}
+** You find the field „Layer“ on top and „Annotation“ below. In the
+*Layer*-dropdown, you can choose the layer you want to annotate with as
+we yust did. You find several default layers linke Part Of Speech or
+Named Entity already. You can also adjust them and create own layers in
+the project settings (we will deal with the project settings later on).
+INCEpTION supports span layers (annotating a span in the text, e.g. from
+the 51th character („letter“) in the text to the 55th) and relation
+layers (annotating two span annotations). For span layers, the default
+is to always annotate one or more tokens („words“) but you can adjust to
+character level or sentence level. For more details, see the main
+documentation. LINK EINFÜGEN.
+** In the field *Annotation* below, you see the details of a selected
+annotation. It shows the layer the annotation is made on (field „layer“)
+and what has been annotated (field „text“). Below, you can see and
+modify what has been entered for each of the so called *Features*. Each
+layer has different features. E.g. on the Part of Speech-layer, the
+PosValue-feature tells if the token is a noun/verb/adjective/… See the
+main documentation for the different feature types and how to adjust
+features for your individual task. LINK EINFÜGEN
++
+You may enter free text here or work with *Tagsets* to have a well
+defined set of values to enter. You can modify and create tagsets in the
+project settings (section „Tagsets“) which we will deal with later. See
+our main documentation on how to create and modify tagsets. LINK
+EINFÜGEN
+* In the **Annotation Sidebar **at the left (it folds out when clicking
+on the little arrow on top) there are several features you might want to
+check the main documentation for. Especially the *Recommender* section
+(the black speech bubble) is worth a look if you use recommenders
+(recommenders will be explained later on in this guide). E.g. you find
+their measures and learning behaviours. Note the *Search* (the magnifier
+glass): You can create or delete annotations on all or some of the
+search results.
+* To get familiar with INCEpTION, you may want to follow the
+instructions for your example-project which are on the website where you
+downloaded it or explore INCEpTION yourself, learning by doing.
++
+One way or the other: *Have fun exploring*!
 
-== Where to go from here
+Anatomy of an annotation project
 
-To familiarize yourself with the functionalities of {product-name}, try importing some of the link:https://inception-project.github.io//use-case-gallery/[{product-name} example projects]. 
+* In order to work with your own projects, you might want to understand
```

Comment:
I would put this section much earlier

Reply:
1. I did so now
2. In the beginning, I write that if someone wants to annotate first, we recommend reading the "first annotations" section first; In the beginning of that section, I write that if someone wants to understand the anatomy first, we recommend reading that first.